roles/config-docker/tasks/kernel_cgroups_v1.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Update Grub config file
+  shell: grubby --args="{{ grub_add_arg }}" --update-kernel=ALL
+
+- name: Reboot the host
+  shell: sleep 5 && shutdown -r now "Ansible Reboot of host"
+  async: 1
+  poll: 0
+  ignore_errors: true
+
+- name: Waiting for the server to come back
```

Comment:
Could be good to consider re-using another role we have for reboot/wait on host - i.e.: https://github.com/redhat-cop/infra-ansible/tree/master/roles/update-host.